scripts/scalalib-patch-tool.sc

```diff
@@ -19,27 +24,26 @@ def main(
     cmd: Command,
     @arg(doc = "Scala version used for fetching sources")
     scalaVersion: String,
-    @arg(
-      doc =
-        "Path to directory containing overrides, defaults to scalalib/overrides-$scalaBinaryVersion"
-    )
-    overridesDir: Option[os.Path] = None
+    // @arg(
+    //   doc =
+    //     "Path to directory containing overrides, defaults to scalalib/overrides-$scalaBinaryVersion"
+    // )
+    // overridesDir: Option[os.Path] = None
```

Comment:
It does not compile/work or it's just commented out by mistake?

Reply:
I commented out for now because it didn't work with scala-cli for some reason, I'll check out in a separate PR 👍